src/examples/forms/radio/RegularWeightLabel.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Radio } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const [radioValue, setRadioValue] = useState('annual');
+
+  return (
+    <Row justifyContent="center">
+      <Col size="auto">
+        <div role="group" aria-label="Choose a plant lifecycle">
+          <Field>
+            <Radio
+              name="weight-example"
+              value="annual"
+              checked={radioValue === 'annual'}
+              onChange={event => setRadioValue(event.target.value)}
+            >
+              <Label isRegular>Annual</Label>
+            </Radio>
+          </Field>
+          <Field>
+            <Radio
+              name="weight-example"
+              value="perennial"
+              checked={radioValue === 'perennial'}
+              onChange={event => setRadioValue(event.target.value)}
+            >
+              <Label isRegular>Perennial</Label>
+            </Radio>
+          </Field>
```

Comment:
Design comment here... Typically a regular weight example would involve adding a `<MD isBold>` heading that summarizes and groups the available choices underneath.

/cc @allisonacs

Reply:
My thinking was that if there is a single `Message` for the radio group, then `8px` spacing wouldn't be needed between the three radio buttons. I could add `8px` spacing between each radio button in this example. Should the other examples (the examples with two radio buttons) also have `8px` spacing?

Reply:
If the radios are butted up against each other, and there are no `hints` or `validation` messages, they don't need space between. It's the addition of optional elements that necessitates the spacing in order for it to look balanced. 

@jzempel is correct, if there is a Radio Group, they should have a heading that is bold, and the radios themselves should have regular weight type.

Reply:
Sounds good, a heading has been added to the validation label.